examples/syscall.c

```diff
@@ -83,37 +115,91 @@ asmlinkage int our_sys_open(const char *filename, int flags, int mode)
 
 static unsigned long **aquire_sys_call_table(void)
 {
-    unsigned long int offset = PAGE_OFFSET;
     unsigned long **sct;
 
+#ifdef HAVE_KSYS_CLOSE
+    unsigned long int offset = PAGE_OFFSET;
+
     while (offset < ULLONG_MAX) {
         sct = (unsigned long **) offset;
 
-        if (sct[__NR_close] == (unsigned long *) close_fd)
+        if (sct[__NR_close] == (unsigned long *) ksys_close)
             return sct;
 
         offset += sizeof(void *);
     }
 
     return NULL;
+#endif
+
+#ifdef HAVE_PARAM
+    char sct_name[15] = "sys_call_table";
+    char addr_symbol[40] = {0};
+
+    if (addr == 0) {
+        pr_alert("There is no input for addr parameter\n");
```

Comment:
Make the message verbose. You should explain.

Reply:
This statement is meaningless. You should improve the writing by check the formal documentation.